Bugzilla/API/V1/User.pm

```diff
@@ -13,8 +13,7 @@ use Mojo::JSON qw( true false );
 
 sub setup_routes {
   my ($class, $r) = @_;
-  $r->get('/api/user/profile')->to('V1::User#user_profile');
-  $r->get('/rest/user/profile')->to('V1::User#user_profile');
+  $r->get('/profile')->to('V1::User#user_profile');
```

Comment:
Won't this end up being `/rest/profile` instead of `/rest/user/profile`?

Reply:
Yeah this is tricky. Having this as /rest/user/profile conflicts with Bugzilla::WebService::User::get in that the older code thinks 'profile' is a user name to return data for and instead errors that 'profile' cannot be found. With #1657 I will be able to properly fix this but for now I have renamed the endpoint to '/rest/user_profile'.